prebuilt-ui/basic-embed/components/Header.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+export const Header = () => (
+  <header className="header">
+    <div className="row">
+      <img src="daily-logo.svg" alt="Daily" className="logo" />
+      <div className="capsule">Prebuilt demo</div>
```

Comment:
this seems to be styled the same as the "API docs" link below. Can you remove the background colour (or something) so they're not styled the same? It makes it look like you can click this otherwise

Reply:
Yes! The thinking here was to match the basic call demo in this repo (screenshot below), but I like the idea of removing. 

![CleanShot 2021-09-14 at 14 51 15](https://user-images.githubusercontent.com/28582074/133338801-cbeef571-b1d9-4272-9693-6a5d7be85dd1.png)